tests/pipeline/avsource/test_gstreamer.py

```diff
@@ -480,3 +484,28 @@ def test_gst_set_state_playing_no_preroll():
     assert not gst.source.is_live
     gst._gst_loop()
     assert gst.source.is_live
+
+
+def test_dev_video_source():
+    
+    def create_gst(fmt=None):
+        gst = GstService(
+            source_conf={
+                "uri": "/dev/video0",
```

Comment:
Does this test past in the cloud CI? Where would it find /dev/video0 ?

Reply:
This test the parsing only, not the actually input. Setting up the transmission [seems feasible](https://stackoverflow.com/questions/18011304/how-to-make-a-virtual-video-device-for-testing-and-debugging-which-has-dev-vide) I think but could require some amount of work